packages/runtime/container-runtime/src/summaryBlobProtocol.ts

```diff
@@ -0,0 +1,128 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * This class represents the object instatiation of the header which is optionaly writen at the
+ * beginning of the binary blob. It supports conversion to / from the binary array.
+ * The binary representation is as follows :
+ * - HEADER ID : 16 bytes of unique identifier : f4e72832a5bd47d7a2f35ed47ed94a3c
+ * - HEADER CONTENT LENGTH : 4 bytes of length of the rest of header content (after the length field)
+ * - HEADER CONTENT :  HEADER CONTENT LENGTH bytes
+ */
+export class BlobHeader {
+    public static readonly ENCODING = "ascii";
+    public static readonly HEADER_PREFIX = Buffer.from("f4e72832a5bd47d7a2f35ed47ed94a3c", "hex");
+    constructor(private readonly _fields: { [key: string]: string; }) { }
+    public getValue(key: string): string {
+        return this._fields[key];
+    }
+
+    /**
+     * This method returns the length of the binary representation of this header object.
+     * @returns Returns the length of the binary representation of this header object.
+     */
+    public headerLength() {
+        return this.toBinary().byteLength;
+    }
+
+    /**
+     * This method converts this header object to binary array.
+     * @returns this header object as binary array.
+     */
+    public toBinary(): Buffer {
```

Comment:
Please use IsoBuffer - leveraging Node's Buffer results in expensive polyfills for browser.

Reply:
Implemented in 2b5e514111440f8ba730e161778193fb9a3ea720.